lib/Models/BingMapsCatalogItem.ts

```diff
@@ -0,0 +1,40 @@
+import Mappable from "./Mappable";
+import CreateModel from "./CreateModel";
+import BingMapsCatalaogItemTraits from "../Traits/BingMapsCatalogItemTraits";
+import BingMapsImageryProvider from "terriajs-cesium/Source/Scene/BingMapsImageryProvider";
+import Credit from "terriajs-cesium/Source/Core/Credit";
+import { computed } from "mobx";
+
+export default class BingMapsCatalaogItem
+    extends CreateModel(BingMapsCatalaogItemTraits)
```

Comment:
Not strictly needed for use as a base map, but because this is named as a catalog item it should really mix in `CatalogMemberMixin`. Also it should have a static and instance `type` property (see `WebMapServiceCatalogItem`).

Also, small typo: BingMapsCatalaogItem